src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,63 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int OPERATION_INDEX = 0;
+    private static final int VALUE_INDEX = 1;
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String fromFile = readFromFileName(fromFileName);
+        String toFile = getResult(fromFile);
+        writeToFile(toFile,toFileName);
+    }
+
+    private String readFromFileName(String fromFileName) {
+        try {
+            BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName));
+            StringBuilder stringBuilder = new StringBuilder();
```

Comment:
For the future, do not use such naming(stringBuilder, toFile, fromFile), how to guess what it stores

Reply:
Today I bought the book Clean Code by Robert Martin, I hope it will help me 
Thanks for the comment